src/code/code_800EC960.c

```diff
@@ -3809,48 +4307,62 @@ s32 func_800F5A58(u8 arg0) {
     }
 }
 
+/**
+ * Plays a sequence on the main bgm player, but stores the previous sequence to return to later
+ * Designed for the mini-boss sequence, but also used by mini-game 2 sequence
```

Comment:
what does "mini-game 2 sequence" refer to?

Reply:
Currently, there are two variants of the minigame sequences that can be played:
https://github.com/zeldaret/oot/blob/79220ba58ae550441e42b4d2c79af7f7e4b9787b/include/sequence.h#L84

and
https://github.com/zeldaret/oot/blob/79220ba58ae550441e42b4d2c79af7f7e4b9787b/include/sequence.h#L114

Eventually, better names will be given to these two to distinguish them. It's just something that I haven't got around to yet. Name suggestions are welcome if you want to look into it! But it'll be addressed eventually

Reply:
This is the sequence that plays during timed minigames.  Think the diving game at Zora's Domain.

Reply:
Oh, that's a simple distinction! So something like `NA_BGM_TIMED_MINI_GAME` could be fine. I wonder if leaving the other one as `NA_BGM_MINI_GAME` is enough or if something extra should be given to it to distnguish it from the "timing of the other one"

Something like `NA_BGM_GENERIC_MINI_GAME` and `NA_BGM_TIMED_MINI_GAME`
(Or `NA_BGM_MINI_GAME_GENERIC` and `NA_BGM_MINI_GAME_TIMED`)